### PR TITLE
[sdks] Add support for building llvm unless the USE_PREBUILT_LLVM Make.config var is set.

### PR DESCRIPTION
**Pull request by vargaz**

### CODE REVIEW

**Comment by vargaz** [comment]
This is required so that xam.ios can be built from source.

File: sdks/builds/ios.mk [hunk 1]
```diff
@@ -297,17 +297,28 @@ $(eval $(call iOSSimulatorTemplate,sim64,x86_64))
 $(eval $(call iOSSimulatorTemplate,simtv,x86_64))
 $(eval $(call iOSSimulatorTemplate,simwatch,i386))
 
-LLVM_REV=3b82b3c9041eb997f627f881a67d20be37264e9c
+ifdef USE_PREBUILT_LLVM
```
Comment (luhenry) [inline sdks/builds/ios.mk]: We have `IGNORE_PACKAGE_LLVM` for XA. It would be useful to use the same variable.

**Comment by vargaz** [comment]
@monojenkins backport 2018-04

**Comment by vargaz** [comment]
@monojenkins backport 2018-02

**Comment by vargaz** [comment]
@monojenkins backport 2017-12

**Pull request merged**